shuup/front/static_src/js/category.js

```diff
@@ -69,6 +69,9 @@ function getFilterString(state) {
 function reloadProducts(filterString) {
     const $cont = $("#ajax_content");
     const $prods = $(".products-wrap");
+    if (!$prods.length) {
```

Comment:
Are you sure this is the issue? The length is tested above and then `$cont.offset().top` is used.

Reply:
Yeah add some more details about the issue. How this works if I add so many filters that there is no products at all? Then how the reload products is supposed to work when I start disabling those filters?